packages/cozy-harvest-lib/test/components/TriggerManager.spec.js

```diff
@@ -432,6 +432,20 @@ describe('TriggerManager', () => {
       await submitPromise
     })
 
+    it('should stop being rendered as submitting on error', async () => {
+      const wrapper = shallowWithAccount()
+      mockVaultClient.isLocked.mockReset().mockImplementationOnce(() => {
+        throw new Error('fakeerror')
+      })
+
+      try {
+        await wrapper.instance().handleSubmit(fixtures.data)
+        expect(mockVaultClient).toHaveBeenCalledTimes(1)
```

Comment:
I don't understand why you check this? Did you want to check that `mockVaultClient.isLocked` has been called?